tinytuya/__main__.py

```diff
@@ -67,6 +66,18 @@
 if force and len(force_list) > 0:
     force = force_list
 
+tinytuya.set_debug(debug, color)
+#if debug:
+#    if color:
+#        logging.basicConfig( format="\x1b[0m\x1b[31;1m%(levelname)s:\x1b[22m%(name)s:\x1b[39;2m%(message)s\x1b[0m", level=logging.DEBUG )
+#    else:
+#        logging.basicConfig( level=logging.DEBUG )
+
+if state == 1:
+    from . import wizard
+else:
+    from . import scanner
```

Comment:
I'm slow or tired or both... why do we want to defer?  Does scanner log logic infect wizard?

Reply:
I'll most likely remove that one and put it back like it was.  I was doing a lot of experimenting and at one point tried to enable debugging before tinytuya was loaded to get the 2 version lines, but it ended up not working and I reverted most of it.